packages/test/package.json

```diff
@@ -21,7 +21,8 @@
     "lint": "eslint src",
     "manualtest": "node dist/test/ManualTests.js",
     "setup": "npx degit https://github.com/LionWeb-io/lionweb-integration-testing/testchanges/data/ data",
-    "test": "mocha ./dist/test/tests.js ./dist/test/inspectionTests.js ./dist/test/tests-issue19.js ./dist/test/history-test.js"
+    "test18": "mocha ./dist/test/tests-issue18.js", 
+    "test": "mocha ./dist/test/tests.js ./dist/test/inspectionTests.js ./dist/test/tests-issue19.js ./dist/test/history-test.js ./dist/test/sequentialize-requests-test.js"
```

Comment:
This seems a bit tedious: can't you use a glob pattern i.c.w. `.skip`-ing tests?

Reply:
There isn't a test18.js anymore, removed this line.